projects/driver_controls/src/main.c

```diff
@@ -136,7 +136,8 @@ int main(void) {
 
   const MechBrakeSettings mech_brake_settings = {
     .ads1015 = &s_pedal_ads1015,
-    .brake_pressed_threshold_percentage = 5,
+    .brake_pressed_threshold_percentage = 55,
```

Comment:
I'm currently using ~70% as a threshold, but we really need to figure out what's causing the calibration to seemingly drift. It might just be that some people brake harder than others.

Reply:
Feel free to adjust the values as you need and then merge it? Doesn't look like we will be resolving/testing this issue before the car leaves.

Reply:
Nah, just merge it. I'll adjust it when I get the chance to debug that.